platforms/linux/hax_wrapper.c

```diff
@@ -40,24 +40,48 @@
 
 #include <asm/cmpxchg.h>
 
-int default_hax_log_level = 3;
 int max_cpus;
 hax_cpumap_t cpu_online_map;
 
-int hax_log_level(int level, const char *fmt,  ...)
+static const char* kLogLevel[] = {
+    KERN_ERR,
+    KERN_DEBUG,
+    KERN_INFO,
+    KERN_WARNING,
+    KERN_ERR,
+    KERN_ERR,
```

Comment:
Should the comma be removed after KERN_ERR?

Reply:
Done